### PR TITLE
Update values for acquisitions

### PR DESCRIPTION
**Pull request by ajlandau**
Update of settings variables for acquisitions to match other samples' settings.

### CODE REVIEW

File: continuous-integration/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 black
 flake8
 pylint
-darglint
\ No newline at end of file
+darglint
+scikit-build
```
Comment (SatjaSivcev) [inline continuous-integration/requirements.txt:5]: What do we need this one for?

File: requirements.txt [hunk 2]
```diff
@@ -3,6 +3,6 @@ netCDF4
 matplotlib
 vtk
 vtk_visualizer
-opencv-python
+opencv-python==4.2.0.34
```
Comment (SatjaSivcev) [inline requirements.txt:6]: This should be a separate commit message with good explanation.

File: source/camera/basic/capture.py [hunk 3]
```diff
@@ -8,9 +8,10 @@ def _main():
     camera = app.connect_camera()
 
     with camera.update_settings() as updater:
-        updater.settings.iris = 40
-        updater.settings.exposure_time = datetime.timedelta(microseconds=40000)
-        updater.settings.filters.reflection.enabled = True
+        updater.settings.iris = 11.31
```
Comment (SatjaSivcev) [inline source/camera/basic/capture.py:11]: The iris is different here compared to the cpp sample. Typo?

https://github.com/zivid/zivid-cpp-samples/blob/master/source/Camera/Basic/Capture/Capture.cpp

Comment (SatjaSivcev) [reply source/camera/basic/capture.py:11]: The iris values are available in the porting guide on the KB.

File: source/camera/basic/capture_hdr_complete_settings.py [hunk 4]
```diff
@@ -17,55 +17,28 @@ def _main():
...
+    for i in range(len(settings_collection)):
+        settings_collection[i].iris = iris_setting[i]
+        settings_collection[i].exposure_time = datetime.timedelta(
+            microseconds=exposure_setting[i]
+        )
+        settings_collection[i].brightness = 1
+        settings_collection[i].gain = gain_setting[i]
+        settings_collection[i].bidirectional = 0
+        settings_collection[i].filters.contrast.enabled = True
+        settings_collection[i].filters.Contrast.threshold = 0.5
```
Comment (SatjaSivcev) [inline source/camera/basic/capture_hdr_complete_settings.py:33]: this should be 5?

File: source/camera/basic/capture_2d.py [hunk 5]
```diff
@@ -8,8 +8,10 @@ def _main():
     camera = app.connect_camera()
 
     settings_2d = zivid.Settings2D()
-    settings_2d.iris = 50
-    settings_2d.exposure_time = datetime.timedelta(microseconds=50000)
+    settings_2d.iris = 11.31
```
Comment (SatjaSivcev) [inline source/camera/basic/capture_2d.py:11]: Zivid-python is based on 1.8 SDK. Therefore we can't use f-numbers but appropriate iris values. This is true for all samples.

File: continuous-integration/requirements.txt (hunk 1 above)
Comment (ajlandau) [reply continuous-integration/requirements.txt]: Removed

File: requirements.txt (hunk 2 above)
Comment (ajlandau) [reply requirements.txt]: This issue was found and fixed in a new PR, approved by Trym.

File: source/camera/basic/capture_2d.py (hunk 5 above)
Comment (ajlandau) [reply source/camera/basic/capture_2d.py]: Fixed

Comment (ajlandau) [reply source/camera/basic/capture_2d.py]: Fixed

**Comment by ajlandau** [comment]
All comments addressed, rebased and updated the commit message. Let me know if any other changes are needed.
